### ISSUE TITLE
sql support lo_creat

### ISSUE THREAD

**Issue by BramGruneir** [cockroachdb/cockroach#26725]
Blob storage currently isn't supported, see #243.

While running the Hibernate tests suite, the following tests require support for lo_creat function.
org.hibernate.userguide.mapping.basic.BlobTest test
org.hibernate.userguide.mapping.basic.BlobByteArrayTest test

Details on the function and how Large Object storage works can be found here:
https://www.postgresql.org/docs/9.6/static/lo-interfaces.html

If we do add blob support, we should consider using the same interface as Postgres.

**Comment by rafiss** [comment]
It comes up in PGJDBC's tests as well: `org.postgresql.test.jdbc4.BlobTest`

**Comment by vy-ton** [comment]
This was on closed boards, should we put in the backlog

**Comment by petergeneric** [comment]
This would be really useful for migrations where PGDUMP has TEXT columns that are stored as large objects

**Comment by shenker** [comment]
I agree that a stub implementation would be really useful for the sake of Postgres API compatibility. I have a legacy Postgres application I can't modify that uses the large object API (even though the objects aren't actually very large and can be stored just fine as normal columns).

**Comment by petergeneric** [comment]
@jlinder / @michae2  are these functions (e.g. `lo_create`, `lo_open`, `lo_write`) something that could be created as stubs using the `CREATE FUNCTION` capabilities currently in CockroachDB? It _seems_ possible, although I've never used functions in CockroachDB before, so a high-level thought on whether it's possible would be useful?

**Comment by michae2** [comment]
> @jlinder / @michae2 are these functions (e.g. `lo_create`, `lo_open`, `lo_write`) something that could be created as stubs using the `CREATE FUNCTION` capabilities currently in CockroachDB? It _seems_ possible, although I've never used functions in CockroachDB before, so a high-level thought on whether it's possible would be useful before I dig in and try to write impls that'd work for basic compat

@petergeneric this would be a better question for @mgartner but I think the answer is something like: it _might_ be possible in v23.2 (currently under development) when mutations in user-defined functions are supported, but definitely not in v23.1 which only supports `SELECT` in UDFs.
